orderer/consensus/kafka/consenter.go

```diff
@@ -52,17 +60,33 @@ func New(config localconfig.Kafka, metricsProvider metrics.Provider, healthCheck
 	}, metrics
 }
 
+// InactiveChainRegistry registers chains that are inactive
+type InactiveChainRegistry interface {
+	// TrackChain tracks a chain with the given name, and calls the given callback
+	// when this chain should be created.
+	TrackChain(chainName string, genesisBlock *cb.Block, createChain func())
+}
+
+type NoopInactiveChainRegistry struct{}
+
+func (*NoopInactiveChainRegistry) TrackChain(chainName string, genesisBlock *cb.Block, createChain func()) {
+}
+
```

Comment:
I think this is not really necessary. We could pass a *nil* chain registry to kafka consenter if it's still operating in kafka mode, or an actual `inactiveChainRegistry` otherwise. So we just need to call `TrackChain` if it's not nil (no type assertion needed). wdyt?

Reply:
makes sense, will adjust.